src/app/shared/date.util.ts

```diff
@@ -102,22 +88,23 @@ export function dateToNgbDateStruct(date?: Date): NgbDateStruct {
  */
 export function dateToString(date: Date | NgbDateStruct): string {
   const dateObj: Date = (date instanceof Date) ? date : ngbDateStructToDate(date);
-
-  let year = dateObj.getUTCFullYear().toString();
-  let month = (dateObj.getUTCMonth() + 1).toString();
-  let day = dateObj.getUTCDate().toString();
-
-  year = (year.length === 1) ? '0' + year : year;
-  month = (month.length === 1) ? '0' + month : month;
-  day = (day.length === 1) ? '0' + day : day;
-  const dateStr = `${year}-${month}-${day}`;
-  return moment.utc(dateStr, 'YYYYMMDD').format('YYYY-MM-DD');
+  return formatInTimeZone(dateObj, 'UTC', 'yyyy-MM-dd');
 }
 
 /**
  * Checks if the given string represents a valid date
  * @param date the string to be checked
  */
 export function isValidDate(date: string) {
-  return moment(date).isValid();
+  return (isNull(date) || isUndefined(date)) ? false : isValid(new Date(date));
```

Comment:
`isNull(date) || isUndefined(date)` can be replaced with `hasNoValue(date)`

Reply:
Thanks @ybnd : I'll get this fixed.